### ISSUE TITLE
【Object Detection API】How to evaluate the validation set on multi-GPUs?

### ISSUE THREAD

**Issue by wangshy31** [tensorflow/models#1977]
【Object Detection API】In terms of training, it is supported for running on multi-GPUs by setting num_clones, but I wonder if it is supported for evaluating the validation set on multi-GPUs. I cannot find related options like "num_clones" in eval.py.
Thank you for providing this API!!

**Comment by drpngx** [comment]
@derekjchow any clue?

**Comment by derekjchow** [comment]
This currently isn't support by the Tensorflow Object Detection API

**Comment by wangshy31** [comment]
Thank you so much! @drpngx @derekjchow

**Comment by ideaplat** [comment]
models/object_detection/config/faster_rcnn_resnet101_pets.config
batch_size: 2
/models/object_detection/train.py
flags.DEFINE_integer(‘num_clones’, 2, ‘Number of clones to deploy per worker.’)

**Comment by wangshy31** [comment]
@ideaplat 
`batch_size: 2 and num_clones=2`
works for training but not validating period
